BenadrylAutomation/src/main/java/com/jnj/pageobjects/ListingPage.java

```diff
@@ -0,0 +1,151 @@
+/**
+ * @author Zahin Ahad
+ * @date 4-March-24
+ */
+
+package com.jnj.pageobjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+
+import com.jnj.actions.Action;
+import com.jnj.base.BaseClass;
+
+public class ListingPage extends BaseClass {
+	Actions actions;
+
+	/**
+	 * Constructor
+	 */
+	public ListingPage() {
+		PageFactory.initElements(driver, this);
+		actions = new Actions(driver);
+	}
+
+	/**
+	 * Locators
+	 */
+	@FindBy(css = "#tab-menu-8651")
+	private WebElement prodHeaderMenu;
+
+	@FindBy(css = "#file-926 img")
+	private WebElement bannerImage;
+
+	@FindBy(css = ".inner > #content-main")
+	private WebElement bannerHeading;
+
+	@FindBy(css = ".inner p")
+	private WebElement bannerSubHeading;
+
+	@FindBy(css = "[data-value='56']")
+	private WebElement adultOralQuickFilter;
+
+	@FindBy(css = "[data-value='711']")
+	private WebElement adultTopicalQuickFilter;
+
+	@FindBy(css = "[data-value='61']")
+	private WebElement kidsProdQuickFilter;
+
+	@FindBy(xpath = "//div[@id='onetrust-close-btn-container']")
+	private WebElement closePrivacyBtn;
+
+	@FindBy(css = ".main-row .three-quarters")
+	private WebElement productsPanel;
+
+	@FindBy(id = "edit-sort-by")
+	private WebElement sortBySelector;
+
+	@FindBy(css = ".sfmc-careclub-lightbox-close")
+	private WebElement closeSignUpBtn;
+
+	private WebElement getBanner(String text) {
+		return driver.findElement(By.xpath("//div[@class='inner']/*[contains(.,'" + text + "')]"));
+	}
+
+	private WebElement getFilter(String text) {
+		return driver.findElement(By.xpath("//div[@class='filter-row-shortcuts']/a[contains(.,'" + text + "')]"));
+	}
+
+	/**
+	 * Function to click on close privacy pop-up button
+	 */
+	public void closePrivacyPopup() {
```

Comment:
is this function used anywhere ? 
@zahinSJI

Reply:
This is used in the spec file

Reply:
I checked in the spec file for Listing. This function is called using the object 'home' and not 'plp' . can you please check if its really used ? and where ?

@zahinSJI